include/IECorePython/WrapperGarbageCollector.h

```diff
@@ -102,7 +102,7 @@ class IECOREPYTHON_API WrapperGarbageCollector
 		/// nor do we want to pay the overhead of garbage collection. Therefore m_pyObject
 		/// is initialised to NULL, isSubclassed() will return false, methodOverride() will always
 		/// fail, and we will have no further overhead.
-		WrapperGarbageCollector( PyObject *self, IECore::RefCounted *wrapped, PyTypeObject *wrappedType );
+		WrapperGarbageCollector( PyObject *wrapperSelf, IECore::RefCounted *wrapped, PyTypeObject *wrappedType );
```

Comment:
The documentation above still refers to `self`. I have to confess to thinking that `self` is a better parameter name though - I don't think we're gaining any clarity here. What is this global `self` that MSVC is so bothered about?

Reply:
I agree, it's a little ugly. The `self` in question is from [boost::python](https://github.com/boostorg/python/blob/6f72675bff276599ffdccb9681018d8b2e367bc2/include/boost/python/self.hpp#L24) as a result of `using namespace boost::python`. Alternatively I could remove the `using namespace boost::python` and add that namespace where needed?

Reply:
Maybe `namespace py = boost::python;` ? Sort of easy to switch to pybind11 in the future... :eyes: